packages/@react-native-windows/automation-commands/src/dumpVisualTree.ts

```diff
@@ -103,3 +108,25 @@ function removeNonDeterministicProps(element: UIElement) {
     element.children.forEach(removeNonDeterministicProps);
   }
 }
+
+/**
+ * Removes noise from snapshot by removing properties with the default value
```

Comment:
I think we should make it clear that it's not because of the specific value of the default value, but that we actually want to filter out *unset* attached properties. It feels like we should update https://github.com/asklar/XamlTreeDump to not return unset attached property values, so we can have that indicator in our snapshots (missing the attached prop rather than showing it with the default value).

Then we wouldn't need to do the filtering here and have to keep the list in sync with the list in the native code. For now can you just change the function name/comment to indicate it's purpose for removing unset attached properties and maybe open/link to a bug on XamlTreeDump?